test/framework/framework.go

```diff
@@ -128,18 +128,25 @@ func (t *TestFramework) CrictlExpectFailureWithEndpoint(
 
 func SetupCrio() string {
```

Comment:
Instead of building cri-o, can we use released binaries? The test should not be testing newest cri-o because it is not cri-o CI.

Reply:
agreed. @saschagrunert could you download a stable release of cri-o in the test?

Reply:
Yes, I’ll do when I’m back in my office in the middle of the next week. :)